internal/executor/executor.go

```diff
@@ -52,6 +52,7 @@ type StepResult struct {
 
 var (
 	ErrStepExit = errors.New("executor step requested to terminate execution")
+	ErrTimedOut = errors.New("timed out")
```

Comment:
Can we not change the error message to avoid changing tests and potentially breaking things for people expecting "Timed out" string in the instruction's logs?

Reply:
Linting will fail I think. Errors should be lowercase

Reply:
Let's add a `//nolint:` then.

Reply:
Can do something like this though [967a6a5](https://github.com/cirruslabs/cirrus-ci-agent/pull/363/commits/967a6a53124977a927455112a0a7ffdf9401b936)